tests/manual/simple_bootloader/Kconfig.sysbuild

```diff
@@ -44,11 +48,18 @@ config NETCORE_APP_UPDATE
 config DFU_MULTI_IMAGE_PACKAGE_NET
 	default y
 
-endif # BOOTLOADER_MCUBOOT
-
 endif # (BOARD_NRF5340DK_NRF5340_CPUAPP || BOARD_THINGY53_NRF5340_CPUAPP)
 
-config PM_EXTERNAL_FLASH_MCUBOOT_SECONDARY
-	default y if BOARD_NRF52840DK_NRF52840 || BOARD_NRF5340DK_NRF5340_CPUAPP || BOARD_THINGY53_NRF5340_CPUAPP
+if (BOARD_NRF54L15DK_NRF54L15_CPUAPP || BOARD_NRF54L15DK_NRF54L15_CPUAPP_NS || BOARD_NRF54L15DK_NRF54L10_CPUAPP)
```

Comment:
use SOC series Kconfig instead

Reply:
Added in next pr: https://github.com/nrfconnect/sdk-sidewalk/pull/659
should be quick to merge, then I rebase this one